index.js

```diff
@@ -2,18 +2,17 @@ let calculator = {
     read(number1, number2) {
         this.Number1 = number1;
```

Comment:
I would suggest to make the Number1 to number1 (same applies for number2), just to follow the naming convention.